src/main/java/seedu/address/model/ReadOnlyMathutoring.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.model;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.student.Student;
+
+/**
+ * Unmodifiable view of an mathutoring
+ */
+public interface ReadOnlyMathutoring {
```

Comment:
Our application still has an addressbook that contains the student contacts. As a developer, I want to know what this class does, what does ReadOnlyMathutoring signifies? I think its better to revert this to ReadOnlyAddressBook

Reply:
Because our project is called Mathutoring, so I think we should change `AddressBook` to `Mathutoring`. The reason of they use `AddressBook` is because their project name is `AddressBook`.